packages/f/freefilesync/pspec_x86_64.xml

```diff
@@ -21,125 +21,32 @@
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/chinese_traditional.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/croatian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/czech.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/danish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/dutch.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/english_uk.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/french.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/german.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/greek.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hebrew.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hindi.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hungarian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/italian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/japanese.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/korean.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/lithuanian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/norwegian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/polish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/portuguese.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/portuguese_br.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/romanian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/russian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/slovak.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/slovenian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/spanish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/swedish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/turkish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/ukrainian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Resources.zip</Path>
+            <Path fileType="data">/usr/share/FreeFileSync/Gtk2Styles.rc</Path>
+            <Path fileType="data">/usr/share/FreeFileSync/Gtk3Styles.css</Path>
+            <Path fileType="data">/usr/share/FreeFileSync/Gtk3Styles.old.css</Path>
+            <Path fileType="data">/usr/share/FreeFileSync/Icons.zip</Path>
+            <Path fileType="data">/usr/share/FreeFileSync/Languages.zip</Path>
```

Comment:
Looks like this is where the langs went, do we need to unzip these to the previous directory or does it auto unzip?

Reply:
No, they are dynamically loaded through wxwidgets own zip archive and reads the files inside.